homeassistant/components/netgear/config_flow.py

```diff
@@ -126,16 +128,17 @@ async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowRes
         device_url = urlparse(discovery_info.ssdp_location)
         if device_url.hostname:
             updated_data[CONF_HOST] = device_url.hostname
-        if device_url.scheme == "https":
-            updated_data[CONF_SSL] = True
-        else:
-            updated_data[CONF_SSL] = False
 
         _LOGGER.debug("Netgear ssdp discovery info: %s", discovery_info)
 
         await self.async_set_unique_id(discovery_info.upnp[ssdp.ATTR_UPNP_SERIAL])
         self._abort_if_unique_id_configured(updates=updated_data)
 
+        if device_url.scheme == "https":
+            updated_data[CONF_SSL] = True
```

Comment:
We don't seem to have a test for this case.

Reply:
My dev enviroment is currently broken, so still trying to get that fixed in order to test this PR.
I followed the instructions here: https://developers.home-assistant.io/docs/development_environment/#developing-on-windows

Unfortunately the lastest Ubuntu version in the windows store only has python version 3.8.10 and HA now requires at least python version 3.9.
If I try to upgrade using `sudo apt update && sudo apt upgrade` and `sudo apt upgrade python3` it will say the latest version is already installed.

If I intall python 3.9 manually using:
```
sudo add-apt-repository ppa:deadsnakes/ppa
sudo apt-get install python3.9
sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 1
sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 2
sudo update-alternatives --config python3
```

I do have python 3.9, but now apt-get is broken, so if you run `sudo apt update && sudo apt upgrade`
you get: `ModuleNotFoundError: No module named 'apt_pkg'`

So I am kind of stuck....

Do you know how to get this working @MartinHjelmare?
Or otherwise maybe add a note on https://developers.home-assistant.io/docs/development_environment/#developing-on-windows that this method is currently broken....

Reply:
Alright got it working using:

```
	sudo apt-get update
	sudo apt update && sudo apt upgrade
	sudo apt upgrade python3
	sudo add-apt-repository ppa:deadsnakes/ppa
	sudo apt-get install python3.9
	sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 1
	sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 2
	sudo update-alternatives --config python3
	sudo apt remove --purge python3-apt
	sudo apt install python3-apt
	curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
	python3.9 get-pip.py
	sudo apt-get install python3-pip python3.9-dev python3.9-venv autoconf libssl-dev libxml2-dev libxslt1-dev libjpeg-dev libffi-dev libudev-dev zlib1g-dev pkg-config libavformat-dev libavcodec-dev libavdevice-dev libavutil-dev libswscale-dev libavresample-dev libavfilter-dev ffmpeg
	script/setup
```